app/models/post.rb

```diff
@@ -21,6 +21,6 @@ def decrement_post_count
   end
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍
- [ ] It would be nice if you make your commit message capitalize to make your pull request descriptive and professional 😊
- [ ] Kindly use a professional picture for your Github Profile because a recruiter will see your picture first and you know the first impression in the last impression so it would be nice if you put a professional picture of yours as the GitHub Profile picture 😊